workflow/rules/diffsplice.smk

```diff
@@ -0,0 +1,92 @@
+rule init_isoform_switch:
+    input:
+        kallisto=kallisto_output,
+        designmatrix="results/sleuth/{model}.designmatrix.rds",
+        fasta="resources/transcriptome.cdna.fasta",
+        gtf="resources/genome.gtf"
+    output:
+        rds="results/diffsplice/{model}.diffsplice.rds",
+        seqs=expand("results/diffsplice/{{model}}.sequences/isoformSwitchAnalyzeR_isoform_{type}.fasta", type=["AA", "nt"])
+    log:
+        "logs/diffsplice/{model}.init.log"
+    conda:
+        "../envs/isoform-switch-analyzer.yaml"
+    params:
+        model=get_model,
+        samples=["{unit.sample}-{unit.unit}".format(unit=unit) for unit in units.itertuples()],
```

Comment:
I think this would conflict with the merging of samples into one file per sample, depending on how we resolve PR #16 .